crates/turborepo-lib/src/boundaries/mod.rs

```diff
@@ -363,7 +381,25 @@ impl Run {
             // Visit the AST and find imports
             let mut finder = ImportFinder::default();
             module.visit_with(&mut finder);
+            let mut warning_count = 0;
             for (import, span, import_type) in finder.imports() {
+                // If the import is prefixed with `@boundaries-ignore`, we ignore it, but print
+                // a warning
+                if Self::is_ignored_import(&comments, *span) {
+                    if warning_count <= MAX_IGNORE_WARNINGS {
```

Comment:
Can we add another case here that outputs the total count of warnings that exceeded the max?